extensions/parquet/table/src/main/java/io/deephaven/parquet/table/pagestore/topage/ToDatePageFromInt.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.parquet.table.pagestore.topage;
+
+import io.deephaven.chunk.ChunkType;
+import io.deephaven.chunk.attributes.Any;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.LocalDate;
+
+import static io.deephaven.util.QueryConstants.NULL_INT_BOXED;
+
+public class ToDatePageFromInt<ATTR extends Any> implements ToPage<ATTR, LocalDate[]> {
+
+    private static final ToDatePageFromInt INSTANCE = new ToDatePageFromInt<>();
+
+    public static <ATTR extends Any> ToDatePageFromInt<ATTR> create(final Class<?> nativeType) {
+        if (nativeType == null || LocalDate.class.equals(nativeType)) {
+            //noinspection unchecked
+            return INSTANCE;
+        }
+
+        throw new IllegalArgumentException("The native type for a Date column is " + nativeType.getCanonicalName());
+    }
+
+    private ToDatePageFromInt() {}
+
+    @Override
+    @NotNull
+    public final Class<LocalDate> getNativeType() {
+        return LocalDate.class;
+    }
```

Comment:
This is correct as-is. At some point we might want to consider whether to keep things as `int` longer, but that's a bigger change.